conjure-undertow-lib/src/main/java/com/palantir/conjure/java/undertow/lib/EndpointRegistry.java

```diff
@@ -27,6 +27,6 @@
  */
 public interface EndpointRegistry {
 
-    EndpointRegistry add(Endpoint description, HttpHandler handler);
+    void add(Endpoint description, HttpHandler handler);
 
```

Comment:
One aspect I like less about this is that now EndpointRegistry cannot be implemented in an immutable fashion anymore.